src/MusicStore/config.json

```diff
@@ -1,4 +1,7 @@
 ﻿{
+    "ApplicationInsights": {
+        "InstrumentationKey": "797d1481-9988-48f2-b41b-f4e305cd9207"
+    },
     "DefaultAdminUsername": "Administrator@test.com",
```

Comment:
How would this work if you don't have a username/password? We need folks to be able to clone Music Store and be able to run it with no changes.